change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=76f07d92bc91a74c6cd53f51c3f29fc2671b18de&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
resolved as noise, the PR does not change anything in the charting components, one of the reported regressions does not look different, the other has some pixels in the chart curve in the diff but does not look to be caused by the files touched in this PR